README.md

```diff
@@ -149,6 +149,18 @@ DJANGO_REST_LOOKUP_FIELD = 'custom_email_field'
 ```
 into Django settings.py file.
 
+## Configurable Response
+
+To configurable response messages need to add following properties into settings.py
+
+```python
+PASSWORD_CHANGED = 'Your password change message' 
+TOKEN_EXPIRED = 'Your token expired'
+PASSWORD_REQUEST_ACCEPT = 'Your password valid request message'
+TOKEN_NOT_FOUND = 'Your token not found message'
+TOKEN_VALID = 'Your token valid message'
```

Comment:
Please use complete and correct english sentences in the example messages. E.g. `Your token has expired`
Users may simply copy/paste these examples into their configuration without realising the full purpose/idea behind it. They should have meaningful and complete messages if that's the case.